README.md

```diff
@@ -9,6 +9,62 @@ Vitrivr NG is a web-based user interface developed to be used with the latest ve
 
 For setup information, consult our [Wiki](https://github.com/vitrivr/vitrivr-ng/wiki)
 
+## Config
+
+There is a `config.json` which enables configuration of the UI to a certain extend.
```

Comment:
This is mostly documented in the wiki: https://github.com/vitrivr/vitrivr-ng/wiki/Configuration already

Reply:
While this is true, I personally like having such information in the README.

Anyways, as described in the PR, once this is merged the corresponding wiki changes will be made (to not have incorrect information in the wiki).

Reply:
That's what the wiki is for - otherwise the README would become very long. We make sure carefully that for new users, vitrivr works out of the box - everything else is for expert users which can consult a Wiki.

Reply:
Exactly, that's why some of the changes included in this PR enhance this.

Reply:
Maybe i added my comment at the wrong line - i'm talking about lines 17 - 67 which belong in the wiki. 14-16 can include a reference to the wiki